recipes/awkward1/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "awkward1" %}
+{% set version = "0.2.37" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/awkward1-{{ version }}.tar.gz
+  sha256: 69a470abe5ac8f9106b90b3753f964ff4654391aecb23ceecac6fffe55c8458b
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33 or py==34]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+  run:
+    - numpy >=1.13.1
```

Comment:
Does this compile and link against numpy? Or is it only used at runtime?

Reply:
The pybind11 code uses

```c++
#include <pybind11/numpy.h>
```

I don't know whether this compiles against NumPy as a C library or if it just sets up the `py::array` conveniences in dealing with a Python buffer object. I suspect the latter, though. If so, then we only use NumPy at runtime (heavily).

Reply:
It's building without having `numpy` in `host` so it should be fine as is.